NewForce-Cohort-9/tabloidmvc-stafford-infection-1
# Delete Function for Categories

# Description

Added functionality for the admin to delete categories. Admin can now navigate to category management where they can view all the categories in alphabetical order and delete any of them after being asked if they are sure they want to delete it.



##  New feature (non-breaking change which adds functionality)

# Testing Instructions
1. Clone down the repository
2. git checkout main
3.  git fetch --all
4. git checkout bb-stafford
5. Open the project (TabloidMVC) in Visual Studio
6. If you haven't already, create a database called "TabloidMVC." The data we are using can be found under the "SQL" folder when looking at our group repository. Be sure to run a query for both files (Create_DB and Seed_Data)
7. Start the server. It should automatically open a webpage on your browser.
8. You'll be greeted by a page that will say "Blogs & Journals", "Get Started", and "Login." Click on Login
9. Login using [admin@example.com] and hit "Submit"
10.  Using the navbar at the top, navigate to "Category Management"
11. Admin will be greeted by the category page where they can see all the categories listed in alphabetical order with buttons beside each one that says "edit", "details", and "delete".
12. When the admin clicks the delete button they are prompted asking them if they are sure they want to delete the chosen category.
13. The admin is presented with a red delete button or a link to go back to the category list.
14. when the delete button is pushed that category is taken off the list.



# Checklist:
